internal/security/config/command/proxy/adduser/command.go

```diff
@@ -266,17 +301,18 @@ func (c *cmd) ExecuteAddJwt() (int, error) {
 	return interfaces.StatusCodeExitNormal, nil
 }
 
+// ExecuteAddOAuth2 (placeholder)
 func (c *cmd) ExecuteAddOAuth2() (statusCode int, err error) {
-	if err := c.createConsumer(); err != nil {
-		return interfaces.StatusCodeExitWithError, err
-	}
-	if err := c.addUserToGroup(); err != nil {
-		return interfaces.StatusCodeExitWithError, err
-	}
```

Comment:
Should this also restore, too?

Reply:
This has been restored.

Reply:
I guess I am not getting it; i still see this change as deleted?

Reply:
Sorry Jim - I fixed the one above - not this one. It has now been corrected and it's coming up in a commit right now...